docker/config/airflow.cfg

```diff
@@ -342,7 +342,7 @@ backend =
 # See documentation for the secrets backend you are using. JSON is expected.
 # Example for AWS Systems Manager ParameterStore:
 # ``{{"connections_prefix": "/airflow/connections", "profile_name": "default"}}``
-backend_kwargs =
+backend_kwargs = ^(?!aws_default$).*$
```

Comment:
This is wrong. Firstly, it is not database backend kwarg, and configuration is different. 
It should be
secrets.backend_kwargs: '{"connections_lookup_pattern":"^(?!aws_default$).*$"}"'
Please refer to this PR : https://github.com/aws/amazon-mwaa-docker-images/pull/68 cc:@kuyperse

Reply:
+ 1 on this. Everything else looks good.

Reply:
> Firstly, it is not database backend kwarg, and configuration is different.
It should be

Yeah, the configuration is actually under the right category (`secrets`), but the diff is making it seems like it is under `database` because it is hiding some lines.

I fixed the value.